charts/ssi-asr/values.yaml

```diff
@@ -62,8 +67,16 @@ migrations:
       cpu: 75m
       memory: 200M
   seeding:
-    testDataEnvironments: ""
-    testDataPaths: "Seeder/Data"
+    # -- Enables dynamic seeding of information related to the operator company: operator.bpn;
+    # If set to `true` the data configured in the config map 'configmap-seeding-initialdata.yaml' will be taken to insert the initial data;
+    useInitial: true
+    # -- Option to seed test data provided in a configMap
+    seedTestData:
+      useOwnConfigMap:
+        # -- ConfigMap containing json files for the tables to seed, e.g. credentials.test.json, authorities.test.json, etc.
+        configMap: ""
+        # -- Filename identifying the test data files e.g. for credentials.test.json the value would be "test"
+        filename: ""
```

Comment:
please remove all seedTestData related part, also in the migrations job under /templates